### DISCUSSION TITLE
Bank-Vaults versioning strategy

### DISCUSSION THREAD

**Discussion by sagikazarmark** [bank-vaults/bank-vaults#2065] [Polls]
During the migration of the project, we started using `v` prefix in version tags in new projects. Unfortunately, the original bank-vaults project doesn't use `v` prefix in version tags which creates understandable confusion (and potential mistakes when tagging a new version for example).

I'm usually all for consistency and not a huge fan of changing versioning schemes, but in this case, it probably makes sense.

We are collecting feedback on what the best approach would be to do that.

**Comment by akijakya** [comment]
Is sticking to the old ways (i.e. dropping the `v`, maybe retag the existing images) not an option? For me the `v` seems to be [unnecessary](https://semver.org/#is-v123-a-semantic-version)...

**Reply by sagikazarmark** [reply]
I generally prefer the prefix, because it clearly signals a release version (tags could be used for all sorts of other purposes).

All of the other projects use it and I'd rather not retag them.
